spec/spec_helper.rb

```diff
@@ -12,6 +12,8 @@
 require 'timecop'
 require 'awesome_print'
 
+require 'pry'
```

Comment:
Whoops, this leaked in.  I'll remove it.  As a side note, is there any good reason *not* to include this in the spec_helper?